notebook.rst

```diff
@@ -1,135 +1,12 @@
 .. _notebook:
    
 ======================  
-The IPython Notebook 
+The Jupyter Notebook (formerly known as IPython notebook)
 ======================
```

Comment:
Have you checked that this still renders as a title? I think rst requires that the over/underline is as long as the text.